src/engine/enginemaster.cpp

```diff
@@ -281,7 +281,7 @@ void EngineMaster::processChannels(int iBufferSize) {
     m_activeChannels.clear();
 
     //ScopedTimer timer("EngineMaster::processChannels");
-    EngineChannel* pMasterChannel = m_pMasterSync->getLeader();
+    EngineChannel* pMasterChannel = m_pMasterSync->getLeaderChannel();
```

Comment:
I'm a bit confused about the `Master` term here. I should rename it anyway (probably in a dedicated PR), but `pMasterChannel` does refer to the *sync leader channel* not the *main output channel*, right? This is a bit confusing.

Reply:
yes, it should be LeaderChannel.  I will file a PR to update the language

Reply:
eh, just fixed it anyway